src/screens/Vault/VaultDetails.tsx

```diff
@@ -235,6 +216,120 @@ function VaultDetails({ navigation, route }: ScreenProps) {
     [isCollaborativeWallet]
   );
 
+  function Footer({
```

Comment:
Why is this moved within the component. Its different component all together. Let's pass the dependency as parms and handle it.